src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -292,259 +300,374 @@ export function DistributionForm({
+                      if (v) return v.decimals;
+                      return 0;
+                    })()}
+                    type="number"
+                    error={!!error}
+                    value={circleDist ? circleDist.gift_amount : value}
+                    disabled={submitting || !!circleDist || vaults.length === 0}
+                    max={Number(maxGiftTokens)}
+                    prelabel={'Budget Amount'}
+                    infoTooltip={
+                      <>
+                        CoVault funds to be allocated to the distribution of
+                        this gift circle.
+                      </>
+                    }
+                    label={`Avail. ${numberWithCommas(
+                      maxGiftTokens
+                    )} ${giftVaultSymbol}`}
+                    onChange={value => {
+                      onChange(value);
+                      setAmount(value);
+                      updateBalanceState(giftVaultSymbol, value, 'gift');
+                    }}
+                    apeSize="small"
+                  />
                 )}
               />
-            </FormControl>
-          </Box>
-          <Box css={{ width: '100%' }}>
-            <Controller
-              name={'amount'}
-              control={control}
-              render={({
-                field: { onChange, value },
-                fieldState: { error },
-              }) => (
-                <ApeTextField
-                  type="number"
-                  error={!!error}
-                  helperText={error ? error.message : null}
-                  value={circleDist ? circleDist.gift_amount : value}
-                  disabled={submitting || !!circleDist}
-                  onChange={({ target: { value } }) => {
-                    onChange(Number(value));
-                    updateBalanceState(
-                      Number(giftVaultId),
-                      Number(value),
-                      'gift'
-                    );
-                  }}
-                  onBlur={({ target: { value } }) => {
-                    setAmount(Number(value));
-                  }}
-                  label={`Available: ${maxGiftTokens}`}
-                  onFocus={event =>
-                    (event.currentTarget as HTMLInputElement).select()
-                  }
-                />
-              )}
-            />
-          </Box>
-        </Box>
-        {!circleDist && (
-          <Box css={{ display: 'flex', justifyContent: 'center' }}>
-            {isCombinedDistribution() ? (
-              <span>
+            </Box>
+          </TwoColumnLayout>
+        </Panel>
+        {(fixedDist || circleDist) && <Summary distribution={circleDist} />}
+        <Flex css={{ justifyContent: 'center', mt: '$xl', mb: '$xl' }}>
+          {!circleDist ? (
+            isCombinedDistribution() ? (
+              <Text css={{ fontSize: '$small', lineHeight: '$xtaller' }}>
```

Comment:
please use padding or fixed-height for the div here, instead of adding this one-off token `$xtaller` to the stitches theme